cmd/neofs-node/config.go

```diff
@@ -532,16 +533,17 @@ var persistateSideChainLastBlockKey = []byte("side_chain_last_processed_block")
 func initCfg(appCfg *config.Config) *cfg {
 	c := &cfg{}
 
-	// attaching version to the node's attributes; do not
-	// move it anywhere below reading the other attributes
-	// since a user should be able to overwrite it.
-	writeAppVersion(c)
-
 	err := c.readConfig(appCfg)
 	if err != nil {
 		panic(fmt.Errorf("config reading: %w", err))
 	}
 
+	// filling system attributes; do not move it anywhere
```

Comment:
So we're not moving it, but we're moving it? IIUC it overwrites now.

Reply:
after [the refactor](https://github.com/nspcc-dev/neofs-node/pull/1830) `readConfig` reads config _literally_ (read numbers/strings and store them in a temporary struct and do not apply them to any component). at least that is what i meant in that PR

applying attributes to a struct that are gonna be used later in `initNetmapService` [func](https://github.com/nspcc-dev/neofs-node/blob/21e9d2acf2f23fd9132b4d3994000d4efca16c9a/cmd/neofs-node/netmap.go#L136)

that is why i changed "do not move it anywhere below reading" to "do not move it anywhere below applying" in the comment